templates/control/edit_apikeys.html

```diff
@@ -78,16 +88,65 @@
         <button class="button negative" style="float: right;">Revoke Selected Consumers</button>
       $else:
         <p style="padding-top: 1em;">You have no OAuth2 consumers!</p>
+    </form>
 
+    <h3>Register OAuth2 Application</h3>
+    <form action="" method="POST" class="clear">
+      $:{CSRF()}
+      <p style="padding-top: 1em;">
+        If you are a developer, you may register an application as an OAuth2 consumer.
+        You can then use your client id and client secret to request authorization tokens from
+        Weasyl. Your client secret MUST be stored securely and should not be accessible
+        by your application's users under any circumstance. For more information see
```

Comment:
I agree with the sentiment, however I'm not sure the phrasing is entirely accurate.
> anyone with your client secret effectively has control over your account
Unless I'm mistaken exposing your client secret would actually allow someone to impersonate your application and use it for malicious purposes without being tracable, but a client secret isn't like API keys in giving any special access to the creator's account.